readme.md

```diff
@@ -1,3 +1,12 @@
 Create a few class that extend each other and show in console initialization order od static, no-static blocks and constructors
```

Comment:
@KYehor I  want you to provide PR with only one commit, Please get a consultation from Ruslan how to have it